src/solver/application/application.cpp

```diff
@@ -128,6 +129,8 @@ void Application::readDataForTheStudy(Data::StudyLoadOptions& options)
             throw Error::NoAreas();
         }
 
+        checkForDuplicates(study);
```

Comment:
Return value is ignored and expect that logs.error() will stop the simulation
Make it void if it's the desired behavior or use the return value